recipes/ptxcompiler/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "ptxcompiler" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/ptxcompiler/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c7021a0946478404c4055a612fd725af95d53ec23eb3dd8d8bc0b5ac55eb44d0
+  fn: ptxcompiler.tar.gz
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
```

```suggestion
  skip: true  # [not linux]
  skip: true  # [cuda_compiler_version == "None"]
  skip: true  # [cuda_compiler_version == "10.2"]
  skip: true  # [cuda_compiler_version == "11.0"]
```

I get confused with too many statements in one line.

I think you can also use syntax like
```
cuda_compiler_version in ["None", "10.2", "11.0"]
```